sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -63,6 +63,7 @@
 
 _DEFAULT_SIZE_FLUSH_THRESHOLD = 10 << 20  # 10MB
 _DEFAULT_TIME_FLUSH_THRESHOLD_MS = 0  # disable time-based flush by default
+_FLUSH_MAX_SIZE = 2 << 30 - 100  # 2GB less some overhead, protobuf/grcp limit
```

Comment:
```suggestion
_FLUSH_MAX_SIZE = (2 << 30) - 100  # 2GB less some overhead, protobuf/grpc limit
```

Reply:
Ah, yeah, that - 100 was a last minute addition.